Rakefile

```diff
@@ -40,7 +40,8 @@ namespace :spec do
 end
 
 desc 'Run all specs'
-task spec: %w[spec:unit spec:integration]
+# TODO: add back integration once we can fix its dep on rules_ios
+task spec: %w[spec:unit]
```

Comment:
What is the issue with the integration tests? I was about to suggest adding a test for this (i.e. something that exercises excluding one platform) but IIUC we're disabling such tests in this PR. Is that correct?

If that's the case can you file an issue and link the issue here so there's more context to fix this in a follow up?

Reply:
filed an issue here: https://github.com/bazel-ios/cocoapods-bazel/issues/94